_maps/RandomZLevels/GUIDELINES.md

```diff
@@ -0,0 +1,24 @@
+# Away Mission Creation Guidelines
+So, you want to make an away mission! They're a lot of fun to play, but they're a challenge to create.
+To ensure a consistent standard of quality across away missions, following the guidelines (split into recommended and required)
+is beneficial to ensure your PR is merged.
+
+## Recommendations
+ * Create a design document
+   * This one is good because it lets you make a coherent thought of what you want your away mission to be, and for others to understand its purpose.
+ * Don't go alone
+   * Away missions are hard work! Making a team of different specializations helps out massively with workload and speed of the project.
+ * Communicate with Maintainers
+   * During the creation of your away mission, it is highly recommended you communicate with maintainers to ensure major changes do not have to be made when it's finished.
+
+## Design Requirements
+ * Minimum average time to complete: 60 minutes
+   * Bare minimum, it should take a team of 4-6 60 minutes to complete.
+ * Design around 4-6 players
+   * Skyrat's gateways use the Vanguard expedition team, which is usually 4 players, but it's best to account for up to 6.
+ * Not purely PvE
+   * Design your gateway with more than just regular simplemobs in mind. Add puzzles, story, PVP, boss fights, etc. to spice up the gameplay instead of a slog of simplemobs.
+ * Loot should not be extreme
+   * The gateway's loot should mostly be given in the purview of using it in the gateway itself (final boss loot and similar not withstanding).
+   * Loot that's unusable outside the gateway is given more leeway.
+   * Loot should not be the primary encouragement for someone to go through the gateway.
```

Comment:
We should really make loot not the entire focus of Away Missions. They shouldn't be balanced nor designed around some insanely stupid thing. Should we add some cool things? Yes, I have no qualms with that. But I'll have qualms if someone adds a "rather-hard-to-get-item" in a rather easy spot. Ideally, I would much prefer if people made/used random spawners to make the mission a tad more random, but only where applicable.

As for boss loot and stuff, it should make sense but still shouldn't be something insane.

Giving you my approval on this with this comment, since I **need** to get a mapping guidelines and stuff written eventually. Nudge me whenever if I don't hold true to my word here.